compiler-core/src/language_server/tests/inlay_hints.rs

```diff
@@ -0,0 +1,72 @@
+use crate::language_server::tests::{setup_engine, LanguageServerTestIO};
+use lsp_types::{InlayHint, InlayHintKind, InlayHintLabel, InlayHintParams, Position, Range};
+
+#[test]
+fn render_inlay_hints() {
```

Comment:
Use snapshots for tests please, and we'll need to be able to change the configuration per test

Reply:
I allowed the tests to have a custom config here:
https://github.com/gleam-lang/gleam/pull/3290/commits/db1162678fa4ab454447d9ef09215b2f2ce50911